docs/environments/simnet.md

```diff
@@ -1,11 +1,16 @@
-# <img class="dcr-icon" src="/img/dcr-icons/Dcrtl.svg" /> Simnet 
+# <img class="dcr-icon" src="/img/dcr-icons/Dcrtl.svg" /> Simnet
 
 ---
 
 When developing Decred applications or testing potential changes, it is often extremely useful to have a test network where difficulty levels are low enough to generate blocks as needed.
 In order to facilitate these scenarios, dcrd provides a simulation network (`--simnet`), where the difficulty starts extremely low to enable fast CPU mining of blocks.
 Simnet also has some modified functionality that helps developers avoid common issues early in development.
 
+As simnet is a private, temporary network for personal use, there are no public
```

Comment:
>As simnet is a private, temporary network for personal use, there are no public...

Is this comma legal? I know replacing periods with commas is the new trend, and I'm starting to do this myself, but it still feels a little wrong :/

Reply:
Reworded these:

> As simnet is a temporary network for private use, there are no public block explorers or faucets available.